python/python/tests/test_db.py

```diff
@@ -610,10 +614,12 @@ async def is_in_v2_mode(tbl):
     assert not await is_in_v2_mode(tbl)
 
     # Create table in v2 mode
-    tbl = await mem_db_async.create_table(
-        "test_v2", data=make_data(), schema=schema, use_legacy_format=False
+    v2_db = await lancedb.connect_async(
+        "memory://", storage_options={"new_table_data_storage_version": "stable"}
     )
 
+    tbl = await v2_db.create_table("test_v2", data=make_data(), schema=schema)
+
```

Comment:
Out of curiosity, what happens if you pass `new_table_data_storage_version` in the `storage_options` parameter of `create_table()`? Does that override what is configured at the connection level?

Reply:
Connection level storage options override any passed to `create_table` (I didn't change this, it's just the way it was)